Source/Core/VideoBackends/D3D/Render.cpp

```diff
@@ -733,11 +733,24 @@ void Renderer::SetDepthState(const DepthState& state)
   m_gx_state.zmode.hex = state.hex;
 }
 
+void Renderer::SetTexture(u32 index, const AbstractTexture* texture)
+{
+  D3D::stateman->SetTexture(
+      index,
+      texture ? static_cast<const DXTexture*>(texture)->GetRawTexIdentifier()->GetSRV() : nullptr);
```

Comment:
I'm not familiar with out D3D helpers, but what shall happen if texture is nullptr? Should we call UnbindTexture instead?